src/confcom/azext_confcom/_params.py

```diff
@@ -267,6 +267,12 @@ def load_arguments(self, _):
             help="Path to a policy fragment to be used with --generate-import to make import statements without having access to the fragment's OCI registry",
             validator=validate_fragment_path,
         )
+        c.argument(
+            "omit_id",
+            options_list=("--omit-id"),
+            required=False,
+            help="Omit the id field in the policy. This is helpful if the image being used will be present in multiple registries and used interchangeably.",
```

Comment:
> This is helpful if the image being used will be present in multiple registries and used interchangeably.
can this also be added to `_help.py`? seems good for the user to know but not sure if it's easily visible to the user currently